static/js/modules/send-message.js

```diff
@@ -6,10 +6,6 @@ var _ = require('underscore'),
 
   'use strict';
 
-  var translateFn = function(key) {
```

Comment:
Was this defined here to avoid some race condition with `exports.init()` being called too late?